consumer/migration/hermes_migrator.go

```diff
@@ -111,6 +120,16 @@ func (m *HermesMigrator) Start(id string) error {
 	if err != nil {
 		return fmt.Errorf("could not get registry address: %w", err)
 	}
+
+	opened, err := m.isChannelOpened(chainID, common.HexToAddress(id), activeHermes, registryAddress)
```

Comment:
what if withdrawal failed but channel was open? I think this blockchain check should be used as some sort of fallback option

Reply:
That is correct. This should be used interchangeably with the transactor call or it should be smarter and assure some withdrawal success.